sbt-plugin/src/sbt-test/testing/multi-framework/test

```diff
@@ -0,0 +1,5 @@
+> root/test
+> multiTestJS/testHtml
+> set scalaJSStage in Global := FullOptStage
```

Comment:
Does this actually *do* anything on its own? Are multiple `>` commands executed within the same session? My impression is that each `>` command reloads sbt, which means that any `set` command on its own is immediately discarded.